docs/index.html

```diff
@@ -778,7 +760,7 @@ <h4 class="headline-3">Module: Grid - Introduction</h4>
 			<div class="grid-cell x3 hint-text">
 				<h4 class="headline-3">Module: Grid - full width</h4>
 				<p>
-					The <code>element</code> class indicates an grid-cell within the grid-box. The grid-system can be divided in 3 grid-cells (width of one grid-cell = x) that can also be combined.
+					The <code>grid-cell</code> class indicates an grid-cell within the grid-box. The grid-system can be divided in 3 grid-cells (width of one grid-cell = x) that can also be combined.
```

Comment:
i would say ..." class indicates a box within the grid"

and add an example for the cells "x1" "x2" "x3"

Reply:
We already have the explanations for them below :)